suite-native/device-manager/src/components/WalletList.tsx

```diff
@@ -0,0 +1,34 @@
+import { useSelector } from 'react-redux';
+
+import { selectDeviceInstances } from '@suite-common/wallet-core';
+import { VStack } from '@suite-native/atoms';
+import { TrezorDevice } from '@suite-common/suite-types';
+
+import { WalletItem } from './WalletItem';
+
+type WalletListProps = {
+    onSelectDevice: (device: TrezorDevice) => void;
+};
+
+export const WalletList = ({ onSelectDevice }: WalletListProps) => {
+    const wallets = useSelector(selectDeviceInstances);
+
+    return (
+        <VStack paddingHorizontal="medium" paddingTop="large">
+            {wallets.map(device => {
+                if (!device.state) {
+                    return null;
+                }
```

Comment:
Not all remembered wallets will have `device.state`. Am I missing something? 🤔

Reply:
device.state is string | undefined

I dont expect state to be undefined here, but for sure (because type allows it) I rather handle this state